### DISCUSSION TITLE
Install agent from invite link - Windows PowerShell

### DISCUSSION THREAD

**Discussion by Raito00** [Ylianst/MeshCentral#5143] [General]
Hello!
Problem to download Agent installation file from invite link.
If i use:
`Invoke-WebRequest -Uri $url -OutFile $dest`

Get error: Invoke-WebRequest : Unauthorized

Looks like download link dont have Auth info ...How can i add Auth to download URI?

**Comment by si458** [comment]
Whats the url you are tying to use?

The invite link isn't a direct url so u won't be able to directly download it

It will take u to a page which you can then get another url to download which version you want
